public/app/plugins/panel/bargauge/panelcfg.cue

```diff
@@ -29,7 +29,7 @@ composableKinds: PanelCfg: {
 					common.SingleStatBaseOptions
 					displayMode:    common.BarGaugeDisplayMode & (*"gradient" | _)
 					valueMode:      common.BarGaugeValueMode & (*"color" | _)
-					valuePlacement: common.BarGaugeValuePlacement & (*"auto" | _)
+					valuePlacement: common.BarGaugeFieldLabelPlacement & (*"auto" | _)
```

Comment:
maybe for consistency we should also rename `valuePlacement`.

Reply:
ah whoops missed this 😬